--- pydhcplib-2b5178f0f85088981e28872343bed25e6825de21/pydhcplib/dhcp_network.py	2012-03-22 00:01:38.000000000 +0100
+++ pydhcplib-2b5178f0f85088981e28872343bed25e6825de21/pydhcplib/dhcp_network.py	2014-06-24 12:32:09.114586110 +0200
@@ -202,9 +202,11 @@
         self.BindToAddress()
 
 class DhcpClient(DhcpNetwork) :
-    def __init__(self, listen_address="0.0.0.0", client_listen_port=68,server_listen_port=67) :
+    def __init__(self, ifname=None, listen_address="0.0.0.0", client_listen_port=68,server_listen_port=67) :
         
-        DhcpNetwork.__init__(self,None,listen_address,client_listen_port,server_listen_port)
+        if ifname!=None:
+            print "Using if=" + str(ifname)
+        DhcpNetwork.__init__(self,ifname,listen_address,client_listen_port,server_listen_port)
 
         self.EnableBroadcast()
         self.EnableReuseaddr()
